monitoring/helm/monitoring/values.yaml.tpl

```diff
@@ -6,12 +6,12 @@ global:
 kube-prometheus-stack:
   grafana:
     namespaceOverride: {{ $monitoringNamespace }}
-{{- if dig "Configuration" "grafana-agent" . }}
+{{- if and .Configuration (index .Configuration "grafana-agent") }}
```

Comment:
perhaps save as a var to reference in both cases more naturally?

Reply:
Done